manifest.json

```diff
@@ -1,4 +1,12 @@
 {
+  "sync.todo": {
+    "name": "Sync Todo App",
+    "repo_owner": "mongodb-university",
+    "repo_name": "realm-template-apps",
+    "backend_path": "todo-sync",
+    "metadata_filename": "Realm",
```

Comment:
If there's no client I don't think we need "metadata_filename" or "file_format". See "Triggers Template" below for an example!